map/search_mark.cpp

```diff
@@ -271,9 +274,10 @@ void SearchMarkPoint::SetFromType(uint32_t type)
   SetAttributeValue(m_type, GetSearchMarkType(type));
 }
 
-void SearchMarkPoint::SetHotelType()
+void SearchMarkPoint::SetHotelType(uint8_t stars)
 {
   SetAttributeValue(m_type, SearchMarkType::Hotel);
+  m_stars = stars;
```

Comment:
That is not necessary if we agree not to provide an unfair advantage to some hotels.